.rubocop.yml

```diff
@@ -75,17 +84,14 @@ Style/FormatString:
 Style/GuardClause:
   Enabled: false
 
-Style/HashSyntax:
-  EnforcedStyle: ruby19
-
 Style/IfUnlessModifier:
   Enabled: false
 
 Style/IndentHash:
   EnforcedStyle: consistent
 
-Style/IndentationWidth:
-  Width: 2
```

Comment:
No need to duplicate the default configuration: https://github.com/bbatsov/rubocop/blob/v0.48.1/config/default.yml#L651-L653